CONTRIBUTING.rst

```diff
@@ -133,13 +133,13 @@ Running System Tests
 
 - To run system tests, you can execute::
 
-   $ nox -s system-3.7
+   $ nox -s system-3.8
    $ nox -s system-2.7
```

Comment:
Please back out changes to this file -- it is generated.

Reply:
Please disregard:  this file is *excluded* from generation, due to Python 2.7 support (see #412).